napari/_qt/qt_viewer.py

```diff
@@ -527,9 +516,10 @@ def console(self, console):
 
     @ensure_main_thread
     def _on_slice_ready(self, event):
+        """This event only gets emitted on async path"""
```

Comment:
Nit: this isn't an event, but it is a callback that is connected to an event.

Reply:
Got it. Will fix.